conf/celeborn-env.sh.template

```diff
@@ -23,8 +23,11 @@
 # - CELEBORN_WORKER_JAVA_OPTS
 # - CELEBORN_PID_DIR
 # - CELEBORN_LOG_DIR
+# - CELEBORN_SSH_OPTS, ssh opts for `start-all` and `stop-all` operations
+# - CELEBORN_SLEEP, Waiting time for `start-all` and `stop-all` operations (e.g. 5, 5s, 5m)
 # - CELEBORN_PREFER_JEMALLOC, to enable jemalloc memory allocator
 # - CELEBORN_JEMALLOC_PATH, to set jemalloc library path
+# - CELEBORN_NO_DAEMONIZE, set `true` to run the proposed command in the foreground
```

Comment:
should we update the value of `CELEBORN_NO_DAEMONIZE` in helm charts to align?

Likes:
https://github.com/apache/celeborn/blob/a357df94f1c1b62aa1d10885f8e815836253e6f0/charts/celeborn/values.yaml#L131

Reply:
@turboFei done